api/generic-cti/generic-cti_no-more-url-varibales.include.rst

```diff
@@ -0,0 +1,8 @@
+🙏 Sorry, but we've decided to remove URL variable samples that were
```

Comment:
IMO we should remove this if we do not document this.

This way looks not good to me.

Personally (from developers view of point) I really like the old examples.

But I can go with not document it at all. But then remove it.